internal/providers/amazon/eks/driver/cluster_creator.go

```diff
@@ -190,11 +192,24 @@ func (c *EksClusterCreator) create(ctx context.Context, logger logrus.FieldLogge
 			Labels:           np.Labels,
 		}
 		asgList = append(asgList, asg)
+
+		nodePoolLabels = append(nodePoolLabels, cluster.NodePoolLabels{
```

Comment:
The `NodePoolLabels ` name is misleading as it seems that beside Labels it holds node pool related fields as well.

Reply:
Actually it holds some fields from node pool like name, instance type & spot price, necessary for cloud info call.